apps/site/assets/ts/schedule/components/schedule-finder/TableRow.tsx

```diff
@@ -189,20 +98,22 @@ const CrTableRow = ({
           <td className="schedule-table__subtable-td">
             <table className="schedule-table__subtable">
               <thead>
-                <TripInfo schedules={schedules} />
+                <TripInfo schedules={trip} />
                 <tr className="schedule-table__subtable-row">
                   <th
                     scope="col"
                     className="schedule-table__subtable-data schedule-table__subtable-data--long"
                   >
                     Stops
                   </th>
-                  <th
-                    scope="col"
-                    className="schedule-table__subtable-data schedule-table__subtable-data--right-adjusted"
-                  >
-                    Fare
-                  </th>
+                  {mode !== 3 && (
```

Comment:
3 == bus. We don't have conversion functions or constants for human readable versions of these in react. In Elixir, we do but generally use the mode number.

Reply:
The magic numbers are slightly less egregious than they might be as they are defined in the GTFS standard (https://developers.google.com/transit/gtfs/reference#routestxt). But since you bring it up let's put a stop to that. I'll make a ticket to replace mode numbers with symbolic constants, both frontend and back.